### PR TITLE
feat: consent updates

### PR DESCRIPTION
**Pull request by foodaka**
## General Changes

- Fixes XYZ bug
- Adds XYZ feature
- …

## Developer Notes

Add any notes here that may be helpful for reviewers.

---

## Author Checklist

Please ensure you, the author, have gone through this checklist to ensure there is an efficient workflow for the reviewers.

- [ ]  The base branch is set to `main`
- [ ]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [ ]  The Github issue has been linked to the PR in the Development section
- [ ]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

**If the PR is ready for review:**

- [ ]  The PR is in `Open` state and not in `Draft` mode
- [ ]  The `Ready for Dev Review` label has been added

## Reviewer Checklist

Please ensure you, as the reviewer(s), have gone through this checklist to ensure that the code changes are ready to ship safely and to help mitigate any downstream issues that may occur.

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  Code changes do not significantly increase the application bundle size
- [ ]  If there are new 3rd-party packages, they do not introduce potential security threats
- [ ]  If there are new environment variables being added, they have been added to the `.env.example` file as well as the pertinant `.github/actions/*` files
- [ ]  There are no CI changes, or they have been approved by the DevOps and Engineering team(s)
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

**Comment by MareskoY** [comment]
Screenshot 2023-06-13 at 13 59 32

need to remove double dots. all antoehr points is ok

**Review by 0x4Graham — APPROVED** [review]

**Comment by grothem** [comment]
I'm running into an issue opting in, not sure if it's related to these changes out not. But if you opt-in, I'm getting an error thrown when trying to track an event, and then the app reloads. On a related note, we should probably wrap the call to track the event in a try/catch so it doesn't crash the app if something goes wrong there. But I'm not sure what would be causing that to happen in this case

File: src/components/cookies/CookieConsent.tsx [hunk 1]
```diff
@@ -4,12 +4,11 @@ import CookieConsentBanner from 'react-cookie-consent';
 import { Link } from 'src/components/primitives/Link';
 import { useRootStore } from 'src/store/root';
 
-import CookieConfigModal from './CookieConfigModal';
-
 export default function CookieBanner() {
```
Comment (grothem) [inline src/components/cookies/CookieConsent.tsx]: I think to save some future confusion we should rename any components or state variables from Cookie -> Analytics.

File: src/components/cookies/CookieConsent.tsx [hunk 2]
```diff
@@ -25,20 +24,23 @@ export default function CookieBanner() {
...
+  const hasUserMadeChoice =
+    typeof window !== 'undefined' && localStorage.getItem('userAcceptedAnalytics') !== null;
+
+  // Note: If they have already chosen don't show again unless configured from footer
+  if (hasUserMadeChoice) return null;
+
   return (
     <>
-      <CookieConfigModal />
+      {/* <CookieConfigModal /> */}
```
Comment (grothem) [inline src/components/cookies/CookieConsent.tsx]: I think this component is dead now and can get removed

**Comment by grothem** [comment]
>

**Comment by grothem** [comment]
Oh I see the issue with event tracking and opting-in. When the app loads, and the user is opted out, or they haven't consented yet, we don't initialize mix panel. If they opt-in, we need to initialize at that point otherwise an error gets thrown when we try to track.

I think we could probably just re-initialize every time the user opts in? Or I suppose we could keep track of if mix panel has been initialized in the store, and only do it if it hasn't been already.

File: src/components/cookies/CookieConsent.tsx (hunk 2 above)
Comment (foodaka) [reply src/components/cookies/CookieConsent.tsx]: removed bbdab4e39cfe9181c05300f77fc342581dca429f

File: src/components/cookies/CookieConsent.tsx (hunk 1 above)
Comment (foodaka) [reply src/components/cookies/CookieConsent.tsx]: done here bbdab4e39cfe9181c05300f77fc342581dca429f

**Comment by foodaka** [comment]
For the initalization I updated it to check in the store. bbdab4e39cfe9181c05300f77fc342581dca429f

**Comment by foodaka** [comment]
@MareskoY i made some updates, can you help with a quick sanity check again?

**Review by grothem — APPROVED** [review]

**Pull request merged**